### PR TITLE
Add composer-require-checker.yml.

### PR DESCRIPTION
**Pull request by terabytesoftw**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,6 +11,42 @@
 
 This package allows us to reuse GitHub actions, which simplifies version management.
 
+### Example of use of the action [Composer require checker](https://github.com/maglnet/ComposerRequireChecker)
+
+```yml
```
Comment (arogachev) [inline README.md:16]: Maybe move this to separate folder and file and refer here with a link? What do you think?

**Review by arogachev — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (arogachev) [reply README.md:16]: Yes, but let's do it in a separate PR.

**Review by xepozz — APPROVED** [review]

File: .github/workflows/composer-require-checker.yml [hunk 2]
```diff
@@ -0,0 +1,55 @@
+name: composer-require-checker
+
+on:
+  workflow_call:
+    inputs:
+      os:
+        description: Os to test, separated by comma.
```
Comment (samdark) [inline .github/workflows/composer-require-checker.yml:7]:
```suggestion
        description: Operating systems to test, separated by comma.
```

File: README.md [hunk 3]
```diff
@@ -11,6 +11,42 @@
 
 This package allows us to reuse GitHub actions, which simplifies version management.
 
+### Example of use of the action [Composer require checker](https://github.com/maglnet/ComposerRequireChecker)
```
Comment (samdark) [inline README.md:14]:
```suggestion
### Using [Composer require checker](https://github.com/maglnet/ComposerRequireChecker)
```

**Comment by samdark** [comment]
👍

File: README.md (hunk 3 above)
Comment (arogachev) [reply README.md]: (Nitpick) Maybe change other headers too to be consistent.

**Pull request merged**